x/posts/internal/types/post.go

```diff
@@ -90,6 +90,17 @@ func (ids PostIDs) Equals(other PostIDs) bool {
 	return true
 }
 
+// AppendIfMissing appends the given post to the list of posts if it does not exist inside it yet,
```

Comment:
```suggestion
// AppendIfMissing appends the given post ID to the list of post IDs if it does not exist inside it yet,
```

Small typo. 

Also, I would change all the documentation to make it more clear. 

```golang
// AppendIfMissing appends the given postID to the ids slice if it doens not exist inside it yet.
// It returns a new slice of PostIDs containing such ID and a boolean indicating whether or not the original 
// slice has been modified. 
```